lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -2727,14 +2727,175 @@ def self.notify
           :allowed_from_server => false,
           :description => 'The port the application is listening on. This setting is mandatory for Passenger servers. Other servers should be detected by default.'
         },
-        :'security.request.body_limit' => {
-          :default => 300,
-          :allow_nil => true,
+        :'security.exclude_from_iast_scan.api' => {
+          :default => [],
+          :public => true,
+          :type => Array,
+          :allowed_from_server => true,
+          :transform => DefaultSource.method(:convert_to_list),
+          :description => 'Defines api paths you want the security agent to ignore in IAST scan, by specifying a list of patterns matching the URI you want to ignore.'
```

Comment:
Are wildcards `*` allows at the end of paths

Reply:
@hannahramadan No, Wild cards are not allowed, The regex pattern should provide a complete match. I have updated the description in with an example in [this](https://github.com/newrelic/newrelic-ruby-agent/pull/2934/commits/df5c3affed130ffeda11308a1b917916f9a1666c) commit.